.gitlab-ci.yml

```diff
@@ -19,10 +21,21 @@ setup virtualenv:
   before_script:
     - export
 
+trigger cvf:
+  stage: .pre
+  variables:
+    # Tell the CVF pipeline to use the current ref of NMODL, this works because
+    # the CVF CI uses the `gitlab-pipelines` helper components.
+    SPACK_PACKAGE_REF_nmodl: "commit='${CI_COMMIT_SHA}'"
+  trigger:
+    project: hpc/cvf
+    # Make the NMODL CI status depend on the CVF CI status
+    strategy: depend
```

Comment:
can we avoid trigger for specific branch like `llvm`?

Reply:
With ecb3b68c4896e50082ad18e9f34935d82da10c2c I added a rule that stops the CVF part being run on PRs to the `llvm` branch. We already only run the CI on PR events, so that should be enough.